### ISSUE TITLE
Release a development version using `cats-effects-1.0.0-RC2`

### ISSUE THREAD

**Issue by bijancn** [cb372/scalacache#191]
Currently, you can't really test out the new `http4s-0.19.0-M1` as the `cats-effect` `IO`s seem incompatible:
```
[error] java.lang.NoSuchMethodError: cats.effect.IO$.ioConcurrentEffect()Lcats/effect/ConcurrentEffect;
[error]         at scalacache.CatsEffect$modes$$anon$2.<init>(CatsEffect.scala:16)                                 
[error]         at scalacache.CatsEffect$modes$.<init>(CatsEffect.scala:15)                        
[error]         at scalacache.CatsEffect$modes$.<clinit>(CatsEffect.scala) 
```

**Comment by cb372** [comment]
I was waiting for cats-effect 1.0 to come out, but there doesn't seem to be any clear timeline on when that will happen.

I'll bump cats-effect and release a snapshot to Sonatype today or later this week.

**Comment by bijancn** [comment]
Super cool! Thanks @cb372

**Comment by sledorze** [comment]
@cb372 Maybe that's the right time now that cats-effect 1.0.0 has landed :)

**Comment by cb372** [comment]
Just released v0.25.0 with support for cats-effect 1.0.0. It's on it's way to Maven Central.
